### PR TITLE
Add irpf2023 1.0

### PR DESCRIPTION
**Pull request by lucasrodcosta**
App for income declaration and tax calculation made for Brazilian tax payers.

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/irpf2023.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  livecheck do
+    url "https://downloadirpf.receita.fazenda.gov.br/irpf/2023/irpf/update/latest.xml"
+    regex(%r{<pkgver>(\d+(\.\d+)+)</pkgver>}i)
+  end
+
+  depends_on macos: ">= :sierra"
+
+  app "IRPF2023.app", target: "IRPF2023 Installer.app"
+  installer manual: "IRPF2023 Installer.app"
```
Comment (reitermarkus) [inline Casks/irpf2023.rb:18]: These two conflict with each other. Only add `installer manual: "IRPF2023.app"`.

File: Casks/irpf2023.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  uninstall rmdir: [
```
Comment (reitermarkus) [inline Casks/irpf2023.rb:20]: This should use `delete`.

**Comment by reitermarkus** [comment]
Is there any reason to still use `irpf2021` or `irpf2022`? If not, those casks should probably be deleted or at least marked discontinued.

File: Casks/irpf2023.rb (hunk 1 above)
Comment (lucasrodcosta) [reply Casks/irpf2023.rb]: Thanks for your help, @reitermarkus. I just made the adjustments here and all tests passed! 🚀

**Comment by lucasrodcosta** [comment]
> Is there any reason to still use `irpf2021` or `irpf2022`? If not, those casks should probably be deleted or at least marked discontinued.

@reitermarkus There is no reason to use `irpf2021` or `irpf2022`.
I'll open a new PR deleting them.

**Review by reitermarkus — APPROVED** [review]

**Pull request merged**